### PR TITLE
MailerLite templates

### PR DESCRIPTION
**Pull request by llacle**
Related to https://github.com/mailerlite/mailerlite/issues/12866

We need to define and publish domain connect templates before proceeding to onboarding with DNS providers. We defined 3 templates, for domain authentication, domain alignment and sites (custom domains).

For security purposes, it is highly recommended that synchronously applied templates with certain variables be digitally signed by the service provider. Domain Connect provides these tools to test and verify signatures: https://exampleservice.domainconnect.org/sig. 
We must publish our public key and place it in a DNS TXT record in a domain specified in the templates in syncPubKeyDomain, it is currently set to `mlsend.com` but can be changed to anything SRE suggests.

### CODE REVIEW

**Comment by nklmilojevic** [comment]
@robgordon89 pls help the team with this next week

**Comment by robgordon89** [comment]
@llacle Think we should keeo this the same as we do on MailerSend https://github.com/Domain-Connect/Templates/blob/master/mailersend.com.mailersend-email.json

can you use mailerlite.com please and use the tool to generate the DNS record etc and then we can add it

**Comment by llacle** [comment]
Hey @robgordon89, we've updated the templates a bit, we included more variables like the MailerSend one, however it's split into 3 templates: 1 for authentication, 1 for alignment and 1 for sites. 

The sites templates has some groupId defined, to allow us to update records depending on what the user selected on the app, for example if they enabled `www` redirect or not.

Let us know if we should merge the alignment and authentication templates into 1.

File: mailerlite.com.website.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+         "type": "CNAME",
+         "host": "www",
+         "pointsTo": "pages.mailerlite.com",
+         "ttl": 3600
+      },
+      {
+         "groupId": "subdomain",
+         "type": "CNAME",
+         "host": "%cnamehost%",
+         "pointsTo": "pages.mailerlite.com",
```
Comment (robgordon89) [inline mailerlite.com.website.json]: can we not pass these as variables too ? like we do for IP it make sense to add varibales to everything (just incase we chnage them later on)

File: mailerlite.com.domain-alignment.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+   "serviceId": "domain-alignment",
+   "serviceName": "MailerLite Domain Alignment",
+   "version": 1,
+   "description": "Control your sender reputation with your own domain.",
+   "syncPubKeyDomain": "mailerlite.com",
+   "variableDescription": "Self explanatory in the variable names.",
+   "records": [
+      {
+         "type": "A",
+         "host": "@",
```
Comment (robgordon89) [inline mailerlite.com.domain-alignment.json:13]: doesnt this need "%inbound_host%"

**Review by robgordon89 — CHANGES_REQUESTED** [review]

File: mailerlite.com.domain-alignment.json (hunk 2 above)
Comment (robgordon89) [reply mailerlite.com.domain-alignment.json:13]: it does not

File: mailerlite.com.website.json (hunk 1 above)
Comment (llacle) [reply mailerlite.com.website.json]: This was updated as variable as requested.

**Comment by llacle** [comment]
Hey @robgordon89, I did some changes. Let me know if I missed something.

**Review by robgordon89 — APPROVED** [review]

**Comment by robgordon89** [comment]
looks good 👍 🥳

**Pull request merged**